core/templates/css/oppia.css

```diff
@@ -1052,6 +1052,11 @@ textarea {
   text-align: center;
 }
 
+.oppia-exploration-tutorial-help-link {
+  font-weight: bold;
+  text-decoration: underline;
+}
+
```

Comment:
Can you add it to the component instead of the global css?

Reply:
Hello @srijanreddy98, I searched for information about "component" since I didn't understand the meaning of it in your comment. It appears that it is related to AngularJS, a framework with which I have no experience. I tried to look up the coding style guide, but the [link](https://github.com/oppia/oppia/wiki/Coding-style-guide#css) for the example code is broken. If you could provide me with an example code to turn it into a component, I would be able to modify my code accordingly. If that's not possible, I'm happy to leave this issue to other contributors. Thank you!

Reply:
Hi @seoeunHong. Sorry for the late response. You can move the highlighted block of code to the end of exploration-editor-tab.component.html inside the `<style>...</style>`. Let me know if you still need help.

Reply:
Hello @srijanreddy98, I really appreciate for your response! I changed the code based on your suggestion. Please let me know if there is any question or some parts to change. Thank you!